docs/data-sources/region.md

```diff
@@ -0,0 +1,42 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
```

Comment:
Be good to wire up https://terraform-docs.io/how-to/github-action/

Reply:
I've added this into #42 as well.